doc/model/train-energy-hessian.md

```diff
@@ -0,0 +1,54 @@
+# Fit energy Hessian {{ pytorch_icon }}
+
+:::{note}
+**Supported backends**: PyTorch {{ pytorch_icon }}
+:::
+
+:::{warning}
+The model trained with Hessian cannot be frozen.
```

Comment:
The model parameters should be able to transferred to a standard energy model, which can be frozen. 

It would be of a great convenient if a user is allow to transfer a standard model to hessian model and vice versa.

Reply:
Thanks for your suggestion. In the new version, it has been stated that "If freezing is forced, the model will be treated as a standard energy model, and the frozen model will no longer be able to output Hessian predictions."